tests/dummy/app/helpers/root-url.ts

```diff
@@ -0,0 +1,16 @@
+import { getOwner } from '@ember/application';
+import Helper from '@ember/component/helper';
+
+export default class RootUrl extends Helper {
+  build(relativeURL: string): string {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore Property `resolveRegistration` does not exist on type `Owner`
+    const ENV = getOwner(this).resolveRegistration('config:environment');
+
+    return `${ENV.rootURL}${relativeURL.replace(/^\//, '')}`;
+  }
+
+  compute([relativeURL]: [string]) {
+    return this.build(relativeURL);
+  }
+}
```

Comment:
Adapted from https://github.com/ef4/ember-root-url/blob/v1.0.1/addon/services/root-url.js

Reply:
We continue to see the screenshots for the compatibility matrix.

<img width="900" alt="" src="https://github.com/ember-intl/ember-intl/assets/16869656/dad9be19-79d3-4a38-bfff-10013e668589">